modules/wayland-proxy.nix

```
@@ -0,0 +1,27 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  options.waylandProxy = {
+    enable = mkEnableOption "Proxied Wayland access" // { default = false; };
```

```suggestion
    enable = mkEnableOption "Proxied Wayland access";
```
`mkEnableOption` defaults to false